govc/vm/clone.go

```diff
@@ -114,6 +118,16 @@ Examples:
   govc vm.clone -vm template-vm new-vm`
 }
 
+func (cmd *clone) searchForSnapshot(snapshotList []types.VirtualMachineSnapshotTree, snapNameToFind string) *types.ManagedObjectReference {
```

Comment:
Can you use the `VirtualMachine.FindSnapshot` instead? https://github.com/vmware/govmomi/blob/17b8c9ccb7f8c7b015d44c4ea39305c970a7bf31/object/virtual_machine.go#L525